dbschema/seeds/017.partnerships.edgeql

```diff
@@ -0,0 +1,83 @@
+with
+partnershipsJson := to_json($$[
+  {
+      "partner": "Rohan Linguistics",
+      "project": "South Downs",
+      "mouStatus": "NotAttached",
+      "financialReportingType": "Funded",
```

Comment:
Only `Managing` type can set this field.
https://github.com/SeedCompany/cord-api-v3/blob/edgedb%2Fbump-gen-lib/src/components/partner/partner.service.ts#L126-L126